lib/workload_mapper.rb

```diff
@@ -44,7 +44,10 @@ def identify_workloads(system_description)
     Dir["#{File.expand_path(workload_mapper_path)}/*.rb"].each do |file_path|
       mapper = WorkloadMapperDSL.new(system_description)
       workload = mapper.check_clue(File.read(file_path))
-      workloads.merge!(workload.to_h)
+      if workload.to_h
+        Machinery::Ui.puts "Detected workload '#{workload.name}'."
+        workloads.merge!(workload.to_h)
+      end
```

Comment:
There should not be output in this class. This can be handle in the `ContainerizeTask` class.